ERROR REPORT DATETIME 2021-06-29 11:50:13.810137-------------------- ERROR IN FILE 96449_doc20210105094602172380 --------------------
[<FrameSummary file C:/Users/Utente/Documents/Multitraccia/Progetti/Cobat/OCR_development/NEW_OCR/main_OCR.py, line 1230 in <module>>, <FrameSummary file C:/Users/Utente/Documents/Multitraccia/Progetti/Cobat/OCR_development/NEW_OCR/main_OCR.py, line 117 in get_dataframe>]
Exception - 'Index' object has no attribute 'count'
